### ISSUE TITLE
Probably typo in Chapter 7 Xgboost

### ISSUE THREAD

**Issue by gelotran** [shokru/mlfactor.github.io#1]
Hi,

Can you check the formula in the part 7.4.3 Xgboost? I think there may be a typo there.

![image](https://user-images.githubusercontent.com/45286621/79588679-9174e280-80d4-11ea-9a0a-82d1ad5e1599.png)

Then the formula 7.5 has to be adjusted accordingly.

Best,
Vu Tran

**Comment by shokru** [comment]
Thanks for the feedback!
I've checked again, and it seems ok to me, given the 2 factor in front of the sum...

**Comment by gelotran** [comment]
Hi, 

Thank for feedback. I take a comparison with XGBoost website.

![image](https://user-images.githubusercontent.com/45286621/79597704-0e0ebd80-80e3-11ea-9679-0ea681e4a5a1.png)

It seems that inside each tree leaf, we sum all the Hessian values  but not a summation of \lamda.  

Btw, your book is great for an R user working with Portfolio ML. I enjoy reading it a lot.

**Comment by shokru** [comment]
True, but I prefer to take the simpler case when the loss is quadratic (easier to follow I find).
And for a quadratic loss, the Hessian is constant. 
That's why I get differences w.r.t to the original derivation with "2" factors at some places.
Thanks for the feedback again: don't hesitate because we really want to continuously improve the book.

**Comment by gelotran** [comment]
You're welcome. Thank for clarification

It's actually a good book. I hope that you can continue to work on it and looking to see further work from you.
